templates/artist/artist_detail.html

```diff
@@ -22,7 +22,7 @@ <h1 class="header-font">{{ artist.name }}</h1>
     <div class="break">
         <hr>
     </div>
-    <img src="{{ artist.photo.img.url }}" />
+    {% include "artist/includes/artist_detail.html" with artist=artist %}
```

Comment:
Yeah, so basically you can replace:
```
{% include "artist/includes/artist_detail.html" with artist=artist %}
```
with
```
{% thumbnail artist.photo.img "500" as thumb %}
    <img src="{{ thumb.url }}" width="{{ thumb.width }}" height="{{ thumb.height }}" alt="{{ artist.name }}" />
{% endthumbnail %}
```

However, you will also need to add:
```
{% load thumbnail %}
```
at the top of the `artist_detail.html` page. This should go immediately after `{% load staticfiles %}`.